src/main/java/com/wcc/platform/utils/PaginationUtil.java

```diff
@@ -0,0 +1,61 @@
+package com.wcc.platform.utils;
+
+import java.util.List;
+import lombok.extern.slf4j.Slf4j;
+
+/** Util class for pagination data for any page. */
+@Slf4j
+public final class PaginationUtil {
+
+  private PaginationUtil() {}
+
+  /**
+   * Get the items for the currentPage with specific pageSize.
+   *
+   * @param items list od data items
+   * @param currentPage page number requested
+   * @param pageSize items per page
+   * @return list of items on the current page
+   */
+  public static <T> List<T> getPaginatedResult(
+      final List<T> items, final int currentPage, final int pageSize) {
+    if (items == null || items.isEmpty()) {
```

Comment:
I would move all this validation to the model using @Min and other annotations. It will make this logic simpler.

Reply:
done